### ISSUE TITLE
Can't reach the last slides

### ISSUE THREAD

**Issue by arthur-fontaine** [SSENSE/vue-carousel#527]
## Can't reach the last slides with `:perPage='3'`and `:scrollPerPage="false"`

(There is already an issue (#345) that explain what happen, but it is closed. The last message of the #345 issue raise the problem, but it seems that the conversation has been forgotten.)

**Current Behavior**
> Cannot reach the last slides by swiping.

**Input Code and steps to reproduce**
```js
<carousel ref="carousel" :paginationEnabled="false" :perPage="3" :scrollPerPage="false">
    ...
</carousel>
```

**Expected behavior/code**
We should be able to reach the last slide.

**Babel Configuration (.babelrc, package.json, cli command)**

```js
{
  presets: [
    '@vue/cli-plugin-babel/preset'
  ]
}
```

**Environment**
- Babel version(s): 7.10.4
- Node/npm version: npm 6.14.5
- OS: macOS 10.15.5

**Additional context/Screenshots**
[A screen recording of the problem](https://github.com/SSENSE/vue-carousel/files/4903611/10.07.20_16_15_43_20-07-10-15-55-26.mp4.zip). Here, it stops at 7. The array of carousel values :
```js
["", 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, ""]
```

**Comment by dennisgeerts** [comment]
I have the same issue. Did you manage to find a solution to this already?

**Comment by arthur-fontaine** [comment]
> I have the same issue. Did you manage to find a solution to this already?

No, I use this instead https://www.npmjs.com/package/vue-awesome-swiper

**Comment by pawel-sz** [comment]
> I have the same issue. Did you manage to find a solution to this already?

it's not a solution, but changing perPage from 5 to 7 helped in my case
